src/main/java/net/minecraftforge/common/BrewingFuelValues.java

```diff
@@ -0,0 +1,61 @@
+package net.minecraftforge.common;
+
+import it.unimi.dsi.fastutil.objects.Object2IntMap;
+import it.unimi.dsi.fastutil.objects.Object2IntOpenHashMap;
+import net.minecraft.world.item.Item;
+
+/**
+ * Stores a map of items to integers representing the fuel value of the items
+ * when used in a brewing stand. If a fuel is not assigned a value, it will use
+ * the fuel value of blaze power. Once added, items cannot be removed from the
+ * fuel values table.
+ */
+public class BrewingFuelValues {
+
+    /**
+     * The value of the only vanilla fuel, blaze powder.
+     */
+    public static final int VANILLA_FUEL_TIME = 20;
+    private Object2IntMap<Item> fuelTimes;
+
+    public BrewingFuelValues() 
+    {
+        this.fuelTimes = new Object2IntOpenHashMap<>();
```

Comment:
As this takes no values from the constructor it can be done in the field initializer.